### PR TITLE
add ability to use ansible-test sanity with docker

### PR DESCRIPTION
**Pull request by richm**
If you set `LSR_ANSIBLE_TEST_DOCKER=true` it will run
`ansible-test sanity --docker`

### CODE REVIEW

**Comment by richm** [comment]
This seems to work a lot better than what we're doing now, which seems broken due to some change in github actions and/or ansible-test

File: src/tox_lsr/test_scripts/runansible-test.sh [hunk 1]
```diff
@@ -64,6 +64,11 @@ for file in "$TOXINIDIR"/.sanity-ansible-ignore-*.txt; do
   fi
 done
 
+if [ "${LSR_ANSIBLE_TEST_DOCKER:-false}" = true ]; then
+  ansible-test sanity --docker 2>&1 | ansible_test_filter
+  exit 0
```
Comment (nhosoi) [inline src/tox_lsr/test_scripts/runansible-test.sh:69]: This means if LSR_ANSIBLE_TEST_DOCKER is true, there is no way to check the result of ansible-test sanity test?

Comment (richm) [reply src/tox_lsr/test_scripts/runansible-test.sh:69]: > This means if LSR_ANSIBLE_TEST_DOCKER is true, there is no way to check the result of ansible-test sanity test?

not sure what you mean - if any of the sanity tests give an error, then `ansible-test` will exit with a non-zero exit code, and due to the use of `set -e`, the script will exit with an error and tox will report the error

Comment (richm) [reply src/tox_lsr/test_scripts/runansible-test.sh:69]: well `set -e` and `-o pipefail`

Comment (nhosoi) [reply src/tox_lsr/test_scripts/runansible-test.sh:69]: :p  You are right.

**Review by nhosoi — APPROVED** [review]
lgtm

**Pull request merged**